plugins/node/opentelemetry-instrumentation-pg/README.md

```diff
@@ -0,0 +1,75 @@
+# OpenTelemetry Postgres Instrumentation for Node.js
+[![Gitter chat][gitter-image]][gitter-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for [`pg`](https://github.com/brianc/node-postgres).
+
+For automatic instrumentation see the
+[@opentelemetry/node](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-node) package.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation-pg
+```
+
+It will install Instrumentation for PG and PG-POOL
+
+## Usage
+
+To load all of the [default supported plugins](https://github.com/open-telemetry/opentelemetry-js#plugins), use the below approach. Each plugin is only loaded when the module that it patches is loaded; in other words, there is no computational overhead for listing plugins for unused modules.
```

Comment:
Should this be re-worded? plugins => instrumentations

Reply:
no, plugins are being loaded not instrumentations, this is fine

Reply:
But why do we mention plugins here? Or is `registerInstrumentation()` without arguments also loading this instrumentation?

Reply:
because until we remove default plugins and remove the plugin loader from registerIntrumentations, this is the only way to load old plugins, not the instrumentations. With instrumentations you have freedom to choose. 
I have created some time ago the tickets to follow up on progress
https://github.com/open-telemetry/opentelemetry-js/issues/1761 - first point done.
https://github.com/open-telemetry/opentelemetry-js/issues/1762
Besides we still didn't agree on creating (and how) or not about meta packages for instrumentations for user to be able to load default instrumentations but that is another topic not here. First we need to move all plugins to instrumentations.